### PR TITLE
Enhanced ops script

### LINKED ISSUE #17: [Enhancement] Update ops script to include all etcd commands and relevant paths
**Feature (What you would like to be added):**
Current script under ops/print-etcd-cert-paths.sh only prints the cert paths. It would be nice to have a list of all paths (data directory, etcd config path, all popular etcdctl commands used by operators) that can just be copied and executed.

**Motivation (Why is this needed?):**
Ease the life of an operator by providing fully formed etcd commands and fully-formed paths to important paths.

### PR DESCRIPTION
**Pull request by unmarshall**
**How to categorize this PR?**

/area dev-productivity
/area ops-productivity
/kind enhancement

**What this PR does / why we need it**:
Introduces a new ops script which will now print all paths and popularly used etcd commands used by operators. Essentially a one stop shot for all things that an operator typically needs/executes.

**Which issue(s) this PR fixes**:
Fixes #17 

**Special notes for your reviewer**:

**Release note**:

```other operator
ops/print-etcd-cert-paths.sh has been removed and is now replaced with ops/print-etcd-cheatsheet.sh
```

### CODE REVIEW

File: ops/print-etcd-cheatsheet.sh [hunk 1]
```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env bash
```
Comment (seshachalam-yv) [inline ops/print-etcd-cheatsheet.sh:1]: license. headers are missing

**Review by seshachalam-yv — CHANGES_REQUESTED** [review]

File: ops/print-etcd-cheatsheet.sh [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+  echo "${process_id}"
+}
+
+function get_backup_restore_process_id() {
+  process_id=$(pgrep etcdbrctl)
+  echo "${process_id}"
+}
+
+wrapper_pid=$(get_wrapper_process_id)
+backup_restore_pid=$(get_backup_restore_process_id)
```
Comment (seshachalam-yv) [inline ops/print-etcd-cheatsheet.sh:14]: Verify that both `wrapper_pid` and `backup_restore_pid` are valid before proceeding

**Comment by aaronfern** [comment]
Thanks for the review @seshachalam-yv!
I've addressed your review comments. PTAL

File: ops/Dockerfile
Comment (shreyas-s-rao) [inline ops/Dockerfile:1]: @aaronfern can you also please update the alpine image version in a separate PR?

**Review by shreyas-s-rao — APPROVED** [review]
/lgtm

**Review by seshachalam-yv — APPROVED** [review]
lgtm

**Pull request merged**